### PR TITLE
Edaman Food API

### PR DESCRIPTION
**Pull request by manisha-27**
Issue no. #2 

1. Nutrition Analysis API
2. Food Database API
3. Recipe Search API

@TheLearneer

### CODE REVIEW

File: src/content/apis/foods/FoodDatabaseAPI.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Edamam
+description: Get free access to a database with close to 900,000 foods and over 680,000 unique UPC codes.
+categories:
+  - foods
+features:
+  https: true
+url: https://developer.edamam.com/food-database-api
+logo: ![small-logo](https://user-images.githubusercontent.com/67019423/193456793-834cee23-1d15-4cfb-8eed-04bdaf733bdb.png)
```
Comment (TheLearneer) [inline src/content/apis/foods/FoodDatabaseAPI.md:9]: The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.

File: src/content/apis/foods/NutritionAnalysisAPI.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+name: Edamam
+description: Do nutrition analysis by Copy/paste any food recipe and learn its nutrition details in under a second
+categories:
+  - foods
+features:
+  https: true
+url: https://developer.edamam.com/edamam-nutrition-api
+logo: ![small-logo](https://user-images.githubusercontent.com/67019423/193456793-834cee23-1d15-4cfb-8eed-04bdaf733bdb.png)
```
Comment (TheLearneer) [inline src/content/apis/foods/NutritionAnalysisAPI.md:9]: The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.

File: src/content/apis/foods/RecipeSearchAPI.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+name: Edamam
+description: Let's you search over 2.3 million recipes by diets, calories and nutrient ranges.
+categories:
+  - foods
+features:
+  https: true
+url: https://developer.edamam.com/edamam-recipe-api
+logo: ![small-logo](https://user-images.githubusercontent.com/67019423/193456793-834cee23-1d15-4cfb-8eed-04bdaf733bdb.png)
```
Comment (TheLearneer) [inline src/content/apis/foods/RecipeSearchAPI.md:9]: https://user-images.githubusercontent.com/67019423/193456793-834cee23-1d15-4cfb-8eed-04bdaf733bdb.png

**Review by TheLearneer — CHANGES_REQUESTED** [review]
HI @manisha-27 Thank you for your contribution.
For the PR to be merged, please go through the requested changes. 🙏

**Review by TheLearneer — CHANGES_REQUESTED** [review]
I just realized that you have 3 instance of same Edamam API, please include only a single file for all those 3.

**Comment by manisha-27** [comment]
Done @TheLearneer

File: src/content/apis/foods/EdamamAPI.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+name: Edamam - Food Database API, Nutrition API and Recipe API
```
Comment (TheLearneer) [inline src/content/apis/foods/EdamamAPI.md:2]:
```suggestion
name: Edamam
```

**Comment by TheLearneer** [comment]
Thank you for your contribution @manisha-27

**Pull request merged**